Bulldozer.CSV/CSVComponent.cs

```diff
@@ -2475,25 +2563,24 @@ private int AddPhoneCountryCodeDVs( RockContext rockContext = null )
         /// <summary>
         /// Add Defined Values for attribute values from csv data that are not yet in Rock
         /// </summary>
-        private int AddAttributeValueDVs( List<AttributeValueObject> attributeValues, RockContext rockContext = null )
+        private int AddAttributeValueDVs( List<AttributeValueObject> attributeValues, RockContext rockContext = null, bool isGroup = false )
         {
             if ( rockContext == null )
             {
                 rockContext = new RockContext();
             }
             var entityTypeIds = new List<int>();
 
-            entityTypeIds.Add( PersonEntityTypeId );
-            entityTypeIds.Add( GroupEntityTypeId );
+            var entityTypeId = isGroup ? GroupEntityTypeId : PersonEntityTypeId;
 
             var definedTypeDict = DefinedTypeCache.All().ToDictionary( k => k.Id, v => v );
             var attributeDefinedTypeDict = new AttributeService( rockContext ).Queryable()
-                                                                                .Where( a => a.FieldTypeId == DefinedValueFieldTypeId && entityTypeIds.Any( e => e == a.EntityTypeId ) )
-                                                                                .ToDictionary( k => $"{k.EntityTypeId}_{k.Key}", v => definedTypeDict.GetValueOrNull( v.AttributeQualifiers.FirstOrDefault( aq => aq.Key == "definedtype" ).Value.AsIntegerOrNull().Value ) );
+                                                                                .Where( a => a.FieldTypeId == DefinedValueFieldTypeId && a.EntityTypeId == entityTypeId )
+                                                                                .ToDictionary( k => $"{k.EntityTypeId}_{k.Key}_{k.EntityTypeQualifierValue}", v => definedTypeDict.GetValueOrNull( v.AttributeQualifiers.FirstOrDefault( aq => aq.Key == "definedtype" ).Value.AsIntegerOrNull().Value ) );
             
             var attributeDefinedValuesDict = attributeDefinedTypeDict.ToDictionary( k => k.Key, v => v.Value.DefinedValues.ToDictionary( a => a.Value, b => b.DefinedTypeId ) );
 
-            var attributeValuesToProcess = attributeValues.Where( v => attributeDefinedValuesDict.ContainsKey( $"{v.EntityTypeId}_{v.AttributeKey}" ) && !attributeDefinedValuesDict[$"{v.EntityTypeId}_{v.AttributeKey}"].ContainsKey( v.AttributeValue  ) )
+            var attributeValuesToProcess = attributeValues.Where( v => attributeDefinedValuesDict.ContainsKey( $"{v.EntityTypeId}_{v.AttributeKey}_{v.EntityTypeQualifierValue}" ) && !attributeDefinedValuesDict[$"{v.EntityTypeId}_{v.AttributeKey}_{v.EntityTypeQualifierValue}"].ContainsKey( v.AttributeValue  ) )
                                                           .Select( v => new AttributeValueObject { AttributeKey = v.AttributeKey, AttributeValue = v.AttributeValue, EntityTypeId = v.EntityTypeId, DefinedTypeId = attributeDefinedTypeDict.GetValueOrNull( $"{v.EntityTypeId}_{v.AttributeKey}" )?.Id } )
```

Comment:
Does this line also need to include the qualifiervalue? I would think so. Be sure to check `attributeDefinedValuesDict` and `attributeValuesToProcess` need to be consistently using the same key value with the qualifiervalue.

```suggestion
                                                          .Select( v => new AttributeValueObject { AttributeKey = v.AttributeKey, AttributeValue = v.AttributeValue, EntityTypeId = v.EntityTypeId, DefinedTypeId = attributeDefinedTypeDict.GetValueOrNull( $"{v.EntityTypeId}_{v.AttributeKey}_{v.EntityTypeQualifierValue}" )?.Id } )
```